### PR TITLE
OCSP Stapling

### PR DESCRIPTION
**Pull request by frju365**
## The problem

- Every time, the browser needs to send a request to LE to check if the SSL Cert is not revoked.

## Solution

- Add OCSP Stapling

## PR Status

Not tested.
Review required.

## How to test

Install Yunohost + nginx

## Validation

- [ ] Principle agreement 0/2 : 
- [ ] Quick review 0/1 : 
- [ ] Simple test 0/1 : 
- [ ] Deep review 0/1 :

### CODE REVIEW

File: data/hooks/conf_regen/15-nginx [hunk 1]
```diff
@@ -54,6 +54,14 @@ do_pre_regen() {
     [[ $main_domain != $domain ]] \
       && touch "${domain_conf_dir}/yunohost_local.conf" \
       || cp yunohost_local.conf "${domain_conf_dir}/yunohost_local.conf"
+    
+    domain_ca_type=$(yunohost domain cert-status ${domain} | grep CA_type | tr -d " ")
```
Comment (Psycojoker) [inline data/hooks/conf_regen/15-nginx]: Now that we have [jq](https://stedolan.github.io/jq/tutorial/) provided with YunoHost, you can do way more cleaner and strong code like this:

```
root@yunorunner:/home/vagrant# yunohost domain cert-status yunorunner.local --json | jq '.certificates."yunorunner.local".CA_type'
"Self-signed"
```

**Comment by frju365** [comment]
Don't know, if it will work as I used just a vm to test condition and not a real one with a signed certificate.

**Comment by frju365** [comment]
Ma PR est correcte ?

**Comment by alexAubin** [comment]
Est-ce que tu l'as testée ? :/

**Comment by alexAubin** [comment]
Proposing to merge in a few days

**Pull request merged**